mmtrack/datasets/transforms/transforms.py

```diff
@@ -350,6 +350,166 @@ def __repr__(self) -> str:
         return repr_str
 
 
+@TRANSFORMS.register_module()
+class CropLikeDiMP(BaseTransform):
+    """Crop images as PrDiMP did.
+
+    The way of cropping an image is proposed in
+    "Learning Discriminative Model Prediction for Tracking."
+    `DiMP <https://arxiv.org/abs/1904.07220>`_.
+
+    Args:
+        crop_size_factor (float): contains the ratio of crop size
+            to bbox size.
+        output_size (float): contains the size of resized image
+            (always square).
+    """
+
+    def __init__(self, crop_size_factor: float, output_size: float):
+        self.crop_size_factor = crop_size_factor
+        self.output_size = output_size
+
+    def crop_like_dimp(
+            self, img: np.ndarray, bbox: np.ndarray, crop_size_factor: float,
+            output_size: int) -> Tuple[np.ndarray, float, np.ndarray]:
```

Comment:
Maybe `[np.ndarray, np.ndarray, float]`? Because in your code, `resize_factor` in the last position